src/icons/file/IconFileAudio.js

```diff
@@ -1,34 +1,49 @@
 // @flow
 import * as React from 'react';
+import uniqueId from 'lodash/uniqueId';
```

Comment:
This is the first of many "file type" icons I updated in this PR. The same update has been applied to all files in `src/icons/file/*` - down to `IconFileZip.js`:

- Converted to a class based component
- Added `outlineId` setup in `componentWillMount()` because each icon instance needs a unique ID for the `filter` attribute to map to.
- Added in the `OutlineFilter`.
- Wrapped the paths in a `<g>` with a filter prop if the `styleProps` are set.

I did the first one manually, and then copied it over to the rest and just updated the pieces that are different from component to component, so if you review this one, the others should be pretty much identical. TODO: in a future PR we should create some kind of `IconFileBase` component and pull out all the shared code into that. There wouldn't be a whole lot left in these files then.